install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,426 @@
+    count: 0
+    profile: 'node_vm'
+  - name: 'lb'
+    count: 0
+    profile: 'node_vm'
+  - name: 'etcd'
+    count: 0
+    profile: 'node_vm'
+  - name: infra
+    count: 0
+    profile: node_vm
+
+# Currently, only all-in-one installation ("openshift_ovirt_all_in_one: true") is supported.
+# Multi-node installation (master and node VMs installed separately) will be supported in a future release.
+openshift_ovirt_all_in_one: true
+openshift_ovirt_cluster: Default
+openshift_ovirt_data_store: data
+openshift_ovirt_ssh_key: "{{ lookup('file', '/root/.ssh/id_rsa_ssh_ocp_admin.pub') }}"
+
+public_hosted_zone:
+# Uncomment to disable install-time checks, for smaller scale installations
+#openshift_disable_check: memory_availability,disk_availability,docker_image_availability
+
+qcow_url: <1>
+image_path: /var/tmp
+template_name: rhelguest7
+template_cluster: "{{ openshift_ovirt_cluster }}"
+template_memory: 4GiB
+template_cpu: 1
+template_disk_storage: "{{ openshift_ovirt_data_store }}"
+template_disk_size: 100GiB
+template_nics:
+  - name: nic1
+    profile_name: ovirtmgmt
+    interface: virtio
+
+debug_vm_create: false
+wait_for_ip: true
+vm_infra_wait_for_ip_retries: 30
+vm_infra_wait_for_ip_delay: 20
+
+node_item: &node_item
+  cluster: "{{ openshift_ovirt_cluster }}"
+  template: "{{ template_name }}"
+  memory: "8GiB"
+  cores: "2"
+  high_availability: true
+  disks:
+    - name: docker
+      size: 15GiB
+      interface: virtio
+      storage_domain: "{{ openshift_ovirt_data_store }}"
+    - name: openshift
+      size: 30GiB
+      interface: virtio
+      storage_domain: "{{ openshift_ovirt_data_store }}"
+  state: running
+  cloud_init:
+    root_password: "{{ root_password }}"
+    authorized_ssh_keys: "{{ openshift_ovirt_ssh_key }}"
+    custom_script: "{{ cloud_init_script_node | to_nice_yaml }}"
+
+openshift_ovirt_vm_profile:
+  master_vm:
+    <<: *node_item
+    memory: 16GiB
+    cores: "{{ vm_cores | default(4) }}"
+    disks:
+      - name: docker
+        size: 15GiB
+        interface: virtio
+        storage_domain: "{{ openshift_ovirt_data_store }}"
+      - name: openshift
+        size: 30GiB
+        interface: virtio
+        storage_domain: "{{ openshift_ovirt_data_store }}"
+      - name: openshift_local
+        size: 30GiB
+        interface: virtio
+        storage_domain: "{{ openshift_ovirt_data_store }}"
+      - name: etcd
+        size: 25GiB
+        interface: virtio
+        storage_domain: "{{ openshift_ovirt_data_store }}"
+    cloud_init:
+      root_password: "{{ root_password }}"
+      authorized_ssh_keys: "{{ openshift_ovirt_ssh_key }}"
+      custom_script: "{{ cloud_init_script_master | to_nice_yaml }}"
+  node_vm:
+    <<: *node_item
+  etcd_vm:
+    <<: *node_item
+  lb_vm:
+    <<: *node_item
+
+cloud_init_script_node: &cloud_init_script_node
+  packages:
+    - ovirt-guest-agent
+  runcmd:
+    - sed -i 's/# ignored_nics =.*/ignored_nics = docker0, tun0 /' /etc/ovirt-guest-agent.conf
```

Comment:
`docker, tun0` should be
`docker tun0`

Reply:
It says `docker0`. Did you want that changed to `docker`?

Reply:
sorry - stupid typo.

Just to remove the comma, so it will be `docker0 tun0`

Reply:
Fixed.